Branch-SDK/Branch-SDK/Networking/Requests/BranchOpenRequest.m

```diff
@@ -99,42 +100,22 @@ - (void)makeRequest:(BNCServerInterface *)serverInterface key:(NSString *)key ca
 }
 
 typedef NS_ENUM(NSInteger, BNCUpdateState) {
-    BNCUpdateStateInstall       = 0,    // App was recently installed.
-    BNCUpdateStateNonUpdate     = 1,    // App was neither newly installed nor updated.
-    BNCUpdateStateUpdate        = 2,    // App was recently updated.
-
-//  BNCUpdateStateError         = 3,    // Error determining update state.
-//  BNCUpdateStateReinstall     = 4     // App was re-installed.
+// Deprecated states, however older SDKs still send them.  The server will ignores these values.
+//    BNCUpdateStateInstall       = 0,    // App was recently installed.
+//    BNCUpdateStateNonUpdate     = 1,    // App was neither newly installed nor updated.
+//    BNCUpdateStateUpdate        = 2,    // App was recently updated.
+//    BNCUpdateStateError         = 3,    // Error determining update state.
+//    BNCUpdateStateReinstall     = 4,    // App was re-installed.
+    
+    BNCUpdateStateUnknown = 0,       // App update status is unknown
```

Comment:
Seems like if an old SDK was sending `BNCUpdateStateInstall` it could potentially get interpreted as `BNCUpdateStateUnknown` since the int value is the same.

Reply:
Also, since all other enum values were deprecated and it is now a two state enum, wouldn't it make sense to drop the use of the enum altogether and just use `[BNCTuneUtility isTuneDataPresent]` wherever the distinction is needed? It would prolly need to be renamed to something like `isFormerTuneUser`.

Reply:
1. Server has been ignoring values 0-4 for some time.  I'm just consolidating them into 0.
2. I can't switch away from enum since the server expects a numerical value in that field.  Even though it is effectively a boolean now.  

This only applies to iOS though.  On android that field is valid.